scripts/components/LviCartItem.ts

```diff
@@ -19,7 +21,7 @@ export default class LviCartItem extends LviCartItemDesign {
         this.btnCartMinus.on(Button.Events.Press, () => {
             this._valueMinus && this._valueMinus();
         });
-        this.lblCloseIcon.on(Label.Events.Touch, () => {
+        this.btnClose.on(Button.Events.Press, () => {
             this._removeValue && this._removeValue();
```

Comment:
We can use null or undefined check like this

```ts
this._removeValue?.();
```

This code will not execute if `this._removeValue` is undefined.

Reply:
DONE.